client/packages/system/src/Stock/ListView/Toolbar.tsx

```diff
@@ -27,6 +27,7 @@ export const Toolbar: FC<{ filter: FilterController }> = () => {
               name: t('label.code-or-name'),
               urlParameter: 'itemCodeOrName',
               placeholder: t('placeholder.enter-an-item-code-or-name'),
+              isDefault: true,
```

Comment:
Okay, this is totally off-topic, but it was annoying me and is consistent with other pages.